traits.css

```diff
@@ -1,64 +1,89 @@
-body{
+* {
+  font-family: 'Lateef', cursive;
+  font-size: 22px;
```

Comment:
guess what im going to say here

Reply:
guess what I'm gonna to reply here :joy: